pages/common/sass.md

```diff
@@ -0,0 +1,20 @@
+# Sass
+
+> Converts SCSS or Sass files to CSS
+> .scss and .sass can be used interchangeably in below examples depending on your preference
+
+- Immediately convert scss file and output css with same filename
+
+`sass {{inputfile}}.scss`
```

Comment:
Oh interesting, my mistake. I think I got mixed up with the functionality of the `--watch` command